hbase-zookeeper/src/main/java/org/apache/hadoop/hbase/zookeeper/ZKUtil.java

```diff
@@ -1400,9 +1399,16 @@ public static void deleteNodeRecursivelyMultiOrSequential(ZKWatcher zkw,
    * Chunks the provided {@code ops} when their approximate size exceeds the the configured limit.
    * Take caution that this can ONLY be used for operations where atomicity is not important,
    * e.g. deletions. It must not be used when atomicity of the operations is critical.
+   *
+   * @param zkw                         ZKWatcher
+   * @param runSequentialOnMultiFailure if true when we get a ZooKeeper exception that could
+   *                                    retry the operations one-by-one (sequentially)
+   * @param ops                         ZKUtilOp list
+   * @throws KeeperException unexpected ZooKeeper Exception
    */
-  static void submitBatchedMultiOrSequential(ZKWatcher zkw, boolean runSequentialOnMultiFailure,
-      List<ZKUtilOp> ops) throws KeeperException {
+  private static void submitBatchedMultiOrSequential(ZKWatcher zkw,
```

Comment:
Please revert this change. We're usually trying to avoid this kind of indentation.